sympy/core/tests/test_evalf.py

```diff
@@ -494,6 +494,19 @@ def test_issue_8853():
     assert get_integer_part(Rational(-1, 2), 1, {}, True) == (0, 0)
 
 
+def test_issue_17681():
+    class identity_func(Function):
+
+        def __new__(cls, *args, **kwargs):
+            return super(identity_func, cls).__new__(cls,  *args, **kwargs)
```

Comment:
I realise that this is just the test code but there are a lot of methods like this around the codebase so I feel I need to point out that this method is not needed.

If you didn't define `__new__` then the superclass `__new__` would have been called anyway so this method has no useful effect.